### DISCUSSION TITLE
Disc Storage for full index

### DISCUSSION THREAD

**Discussion by catch-21** [CardanoSolutions/kupo#6] [Q&A] [answered]
Hey, I have a mainnet instance with no address filter (`--match "*"`) and the db is almost 24GB. I was wondering what the "4GB (full index)" in System Requirements was referring to?

**Comment by KtorZ** [comment, marked as answer]
It's referring to the testnet, and a gross approximation / extrapolation of what the mainnet would be and, it was also before I had to alter the database schema for a feature, causing the whole database to grow about twice as large as initially measured (yet, with the benefits of reducing underlying code complexity; storage is relatively cheap so I believe this is an okay trade-off!). 

If you have the exact number, I'd be glad to update the documentation. On the plus side, the memory requirements seem also to be a lot less than 512MB in practice, which is great.

**Reply by catch-21** [reply]
I see you've updated the documentation already. I am seeing 34GB with a full mainnet sync with no filters so 40GB should future proof for a while for most ::)
